src/metalperformanceshaders.cs

```diff
@@ -3387,6 +3542,10 @@ interface MPSImageAllocator : NSSecureCoding {
 		[Abstract]
 		[Export ("imageForCommandBuffer:imageDescriptor:kernel:")]
 		MPSImage GetImage (IMTLCommandBuffer cmdBuf, MPSImageDescriptor descriptor, MPSKernel kernel);
+
+		[iOS (11,3), TV (11,3), Mac (10,13,4, onlyOn64: true)]
+		[Export ("imageBatchForCommandBuffer:imageDescriptor:kernel:count:")]
+		NSArray<MPSImage> GetImageBatch (IMTLCommandBuffer cmdBuf, MPSImageDescriptor descriptor, MPSKernel kernel, nuint count);
```

Comment:
Parameter names are technically a part of the API, so we prefer to not use abbreviations:

```suggestion
		NSArray<MPSImage> GetImageBatch (IMTLCommandBuffer commandBuffer, MPSImageDescriptor descriptor, MPSKernel kernel, nuint count);
```